File: src/main/java/com/refinedmods/refinedstorage/util/PacketBufferUtils.java
package com.refinedmods.refinedstorage.util;

import net.minecraft.network.FriendlyByteBuf;

public final class PacketBufferUtils {
    private PacketBufferUtils() {
    }

    // @Volatile: From PacketBuffer#readUtf, this exists because SideOnly
    public static String readString(FriendlyByteBuf buffer) {
        return buffer.readUtf(32767);
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/util/PacketBufferUtils.kt
package com.refinedmods.refinedstorage.util

import net.minecraft.network.PacketByteBuf

object PacketByteBufUtils {
    // @Volatile: From PacketByteBuf#readString, this exists because SideOnly
    fun readString(buffer: PacketByteBuf): String {
        return buffer.readString(32767)
    }
}